.vale/styles/Localization/PhrasalVerbs.yml

```diff
@@ -0,0 +1,18 @@
+---
+extends: substitution
```

Comment:
Per Aidan's suggestion, we can add these that Gaurav thought of: @Chandralekha-RedHat 

call up
carry on
carry out
click on
find out
print out
start up
switch on
tap on

Reply:
Aidan also suggested:
"Also maybe, "picks? up," "gives? up," or "sets? up""

I am fine with 
picks up
gives up

But I do NOT think we should add "sets up" because set-up is a valid term.

Reply:
Thanks! Updated as suggested!